### ISSUE TITLE
Search on tag page is broken

### ISSUE THREAD

**Issue by VictorZakharov** [zzossig/hugo-theme-zzo#279]
I updated to latest code from zzo upstream, and hugo version to match.

Using my own branch to add build timestamp and remove all tags from cloud that are only used once. Aside from that, I don't have any customizations. FF merge worked, no conflicts.
https://github.com/VictorZakharov/hugo-theme-zzo

Search now works on main page
https://www.neolisk.blog/

but not on tag page:
https://www.neolisk.blog/tags/algorithm/

Do you have an idea?

**Comment by VictorZakharov** [comment]
![image](https://user-images.githubusercontent.com/8978766/83073496-89e31a80-a03e-11ea-9710-8da4286daa5f.png)

**Comment by VictorZakharov** [comment]
I have a fix.
```
(function initFuse() {
      var xhr = new XMLHttpRequest();
      xhr.open('GET', "/index.json");
```

**Comment by zzossig** [comment]
I have fixed this issue with this commit f321503b94fafa405666acfec7c574ae50c60323
